common/src/main/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsTracker.kt

```diff
@@ -14,6 +16,42 @@ class DiagnosticsTracker(
     private val diagnosticsDispatcher: Dispatcher
 ) {
 
+    fun trackHttpRequestPerformed(
+        endpoint: Endpoint,
+        responseTime: Long,
```

Comment:
Does Kotlin have any sort of "Duration" type? It would be nice to encode in the type system that this is milliseconds.

Reply:
Good idea! There is a `Duration` type indeed representing a different in time. I think that could fit here well 👍